crt_portal/cts_forms/sorts.py

```diff
@@ -6,14 +6,17 @@
 SORT_DESC_CHAR = '-'
 
 
-def _valid_sort_params(sort):
-    valid_fields = [f.name for f in Report._meta.fields] + [f.name for f in EmailReportCount._meta.fields]
+def _valid_sort_params(sort, type):
+    if type == 'activity':
+        valid_fields = ['timestamp', 'verb', 'description', 'target_object_id']
+    else:
+        valid_fields = [f.name for f in Report._meta.fields] + [f.name for f in EmailReportCount._meta.fields]
```

Comment:
nit: one way of doing this that drops for `f.name for f in` duplication:

```suggestion
        fields = [
          *EmailReportCount._meta.fields,
          *Report._meta.fields,
        ]
        valid_fields = [f.name for f in fields]
```